pyblish/api.py

```diff
@@ -151,6 +151,11 @@ def __init__():
         if host:
             register_host(host)
 
+    # Register targets for current session
+    for t in os.environ.get("PYBLISH_TARGETS", "").split(os.pathsep):
```

Comment:
This is too short of a variable name.

Can we also make it..

```python
if not target:
  continue

register_target(target)
```

To reduce indentation?

Reply:
Alternatively..

```python
register_target(target) if target else None
```

Reply:
If I do that then I want to change the lines (line 150) above it as well to have consistency.
I used this as a template to ensure the logic would be the same:
```python
# Register hosts from environment "PYBLISHHOSTS"
for host in os.environ.get("PYBLISH_HOSTS", "").split(os.pathsep):
    if host:
        register_host(host)
```

These lines would then become:
```python
# Register hosts from environment "PYBLISHHOSTS"
for host in os.environ.get("PYBLISH_HOSTS", "").split(os.pathsep):
    if not host:
        continue
    register_host(host)
```

Agreed?